main/src/ca/uwaterloo/flix/api/lsp/LanguageServer.scala

```diff
@@ -98,6 +100,38 @@ class LanguageServer(port: Int, o: Options) extends WebSocketServer(new InetSock
     */
   private var currentErrors: List[CompilationMessage] = Nil
 
+  /**
+    * The java classes in the jdk classList
+    */
+  private val javaClasses: MultiMap[List[String], String] = sys.env.get("JAVA_HOME") match {
+    case None => MultiMap.empty
+    case Some(home) =>
+      val path = s"$home/lib/classlist"
```

Comment:
I think here we have to be very careful. First `/` is not the right separator on all platforms. 

I think we should construct a Path object.

Then we should ensure that it is: (1) exists, (2) is a regular file, and (3) is readable.